apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/Indictment.tsx

```diff
@@ -234,7 +279,10 @@ const Indictment = () => {
         return
       }
 
-      setDriversLicenseSuspensionRequest(
+      const prevSuspensionOffences = getSuspensionOffences(
+        workingCase.indictmentCounts,
+      )
+      const newSuspensionOffeenses = getSuspensionOffences(
```

Comment:
Small typo.

```suggestion
      const newSuspensionOffences = getSuspensionOffences(
```

Additionally, we mostly use the US spelling of "Offense" rather than "Offence" so maybe it should be changed to the US spelling for consistency (though I personally actually prefer "Offence" 😄)

Reply:
We should be consistent, so using offense is an easier change.